torchsde/_core/adjoint_sde.py

```diff
@@ -58,32 +58,72 @@ def __init__(self,
             NOISE_TYPES.diagonal: self.gdg_prod_diagonal,
         }.get(sde.noise_type, self.gdg_prod_default)
 
-    def _unpack_y_aug(self, y_aug):
+    def _get_state(self, t, y_aug, v):
```

Comment:
The name `_get_state` is not very meaningful. I'd either change this to something more meaningful, or include a one-liner docstring header explaining its purposes.

Reply:
Added docstring, and trimmed the comment a little.